storage/remote/read_test.go

```diff
@@ -350,7 +350,7 @@ func TestSampleAndChunkQueryableClient(t *testing.T) {
 			mint: 1, maxt: 2,
 			matchers: []*labels.Matcher{
 				labels.MustNewMatcher(labels.MatchNotEqual, "a", "something"),
-				// TODO(bwplotka): Is this what we want? I prometheus is region=europe and user requests region=us we
+				// TODO(bwplotka): Is this what we want? If prometheus is in region=europe and user requests region=us we
```

Comment:
Yes, this is the desired behaviour. US&Europe isn't very sane as they're at the same level, but think about a hierarchical setup.

Reply:
ok